sensio/sphinx/codeblock.py

```diff
@@ -0,0 +1,16 @@
+from sphinx.directives.code import CodeBlock
+
+"""
+The only change with the built-in CodeBlock class that comes with Sphinx
+is that linenos is always on.
+"""
+class NumberedCodeBlock(CodeBlock):
+    def run(self):
+        literal = super(NumberedCodeBlock, self).run()[0];
+        literal['linenos'] = 'linenos'
```

Comment:
Yeah, true is supported by Pygments 0.7 - 2.0 (for BC reasons). Not sure if we want that.